wpilibNewCommands/src/main/native/include/frc2/command/Requirements.h

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#pragma once
+
+#include <initializer_list>
+#include <span>
+#include <vector>
+
+#include "frc2/command/Subsystem.h"
+
+namespace frc2 {
+
+/**
+ * Represents requirements for a command, which is a set of (pointers to)
+ * subsystems. This class is implicitly convertible from std::initializer_list
+ * and std::span, allowing AddRequirements({&subsystem1, &subsystem2}). As a
+ * note of caution, however, AddRequirements({&subsystem}) selects the
+ * AddRequirements(Command*) overload, which may lead to warnings about the
```

Comment:
I'm not sure if this documentation is needed/belongs here- AddRequirements is part of `Command`

 Also, when you use code in doxygen documentation, wrap in `{@code }` or `\code \endcode`